android/app/src/main/java/org/bitcoinppl/cove/flow/new_wallet/cold_wallet/ColdWalletQrScanScreen.kt

```
@@ -0,0 +1,143 @@
+                    } catch (e: WalletException.WalletAlreadyExists) {
+                        try {
+                            app.rust.selectWallet(id = e.v1)
+                            app.popRoute()
+                            app.alertState =
+                                TaggedItem(
+                                    AppAlertState.General(
+                                        title = "Success",
+                                        message = "Wallet already exists: ${e.v1}",
+                                    ),
+                                )
+                        } catch (selectError: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)